Wikipedia/Code/WMFArticleViewController.m

```diff
@@ -98,8 +98,16 @@ - (nullable UIActivityViewController *)sharingActivityViewControllerWithTextSnip
     if (url) {
         [items addObject:[[NSURL alloc] initWithString:[NSString stringWithFormat:@"%@?%@", url.absoluteString, @"wprov=sfsi1"]]];
     }
+    
+    if (CLLocationCoordinate2DIsValid(self.coordinate)) {
```

Comment:
Would probably be safer to create and assign the MKMapItem to a var, then check that it != nil before adding it to the array.